Update documentation links (#631)
Princeton-LSI-ResearchComputing/tracebase: TraceBaseDocs/TraceBaseDocs/Types of Data Output/Fcirc.md

# Fcirc

## Definition

Circulatory Flux (Fcirc) is defined as the steady-state flux of metabolite
between tissues and bloodstream(see [Bartman, TeSlaa, and
Rabinowitz](https://doi.org/10.1038/s42255-021-00419-2).  Typically, Fcirc is
measured by infusing a labeled tracer to a steady state and measuring its
dilution in the animal.  The labeling of the tracer compound in serum and the
known rate of tracer infusion are used to calculate Fcirc.  There are key
assumptions of steady state and minimal perturbation in typical calculations of
Fcirc - see the above reference for details.

## Types of Fcirc

A researcher may wish to calculate multiple forms of Fcirc depending on their
experimental goals.  These forms are based on whether the tracer infusion is
included in the flux (Ra vs Rd) and whether the intact or atom-averaged flux is
preferred.  TraceBase calculates all of four of these.

![image](https://user-images.githubusercontent.com/34348153/205693110-8e852f8f-0c27-456e-a42c-e705e42ff72a.png)
(from Bartman, TeSlaa, and Rabinowitz)

TraceBase automates the standard calculations for Fcirc without making any

> assumptions.  Note that **TraceBase attempts to calculate Fcirc for every
> serum/plasma sample, so it will sometimes estimate Fcirc that is invalid (ie
> before steady state)**.  In other cases, TraceBase will report an Ra value in
> an infusion that was perturbative.  This can still be a useful and valid
> measurement, but it should not be interpreted as the normal circulatory flux
> (Fcirc).

Reviewer: nb. Slight rewording suggestion for precision.
```suggestion
assumptions.  Note that **TraceBase attempts to calculate Fcirc for every
serum/plasma sample, so be aware that some estimates displayed are invalid (ie
before steady state)**.  In other cases, TraceBase will report an Ra value in
an infusion that was perturbative.  This can still be a useful and valid
measurement, but it should not be interpreted as the normal circulatory flux
(Fcirc).
```
